.github/workflows/release.yml

```diff
@@ -2,7 +2,7 @@ name: release
 on:
   push:
     tags:
-      - 'v*'
+      - 'v*.*'
```

Comment:
should this be `v.*` ... or do you really want the `v` to be optional?

Reply:
i think this is a glob not a re.... let me double check that

Reply:
Yup it's glob https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet